lib/run/index.js

```diff
@@ -86,6 +86,11 @@ module.exports = function (options, callback) {
             globals: options.globals,
             entrypoint: options.folder,
             data: options.iterationData,
+            delay: {
+                item: options.delayRequest
+            },
+            suppressExitCode: options.suppressExitCode,
```

Comment:
This will have no effect, because Runtime doesn't know anything about exit codes. It's a newman-level option :)